server/modules/surveyRdb/repository/dataView/read.js

```diff
@@ -12,31 +12,133 @@ import * as Expression from '../../../../../core/expressionParser/expression'
 
 import * as SchemaRdb from '../../../../../common/surveyRdb/schemaRdb'
 import * as NodeDefTable from '../../../../../common/surveyRdb/nodeDefTable'
-
 import * as DataSort from '../../../../../common/surveyRdb/dataSort'
 import * as DataFilter from '../../../../../common/surveyRdb/dataFilter'
-import { ViewDataNodeDef } from '../../../../../common/model/db'
+import { ViewDataNodeDef, TableNode, TableResultNode, ColumnNodeDef } from '../../../../../common/model/db'
 import { getSurveyDBSchema } from '../../../survey/repository/surveySchemaRepositoryUtils'
+import { dbTransformCallback as dbTransformCallbackNode } from '../../../record/repository/node'
```

Comment:
a repository cannot use another repository. add this method to tableNode

Reply:
ok